plugins/ltac/tacinterp.ml

```diff
@@ -1108,10 +1101,6 @@ and eval_tactic ist tac : unit Proofview.tactic = match tac with
       in
       Ftactic.run msg tac
   | TacProgress tac -> Tacticals.New.tclPROGRESS (interp_tactic ist tac)
-  | TacShowHyps tac ->
-         Proofview.V82.tactic begin
-           tclSHOWHYPS (Proofview.V82.of_tactic (interp_tactic ist tac))
```

Comment:
If this Ltac ends up being removed, you should also remove `tclSHOWHYPS` from `Refiner`, since it would be dead code.